flink/v1.17/flink/src/test/java/org/apache/iceberg/flink/source/TestFlinkScan.java

```diff
@@ -49,37 +51,28 @@
 import org.apache.iceberg.types.Types;
 import org.apache.iceberg.util.DateTimeUtil;
 import org.assertj.core.api.Assertions;
-import org.junit.Assert;
-import org.junit.ClassRule;
-import org.junit.Rule;
-import org.junit.Test;
-import org.junit.rules.TemporaryFolder;
-import org.junit.runner.RunWith;
-import org.junit.runners.Parameterized;
-
-@RunWith(Parameterized.class)
```

Comment:
I would hold off on changing any parameterized tests until we have a JUnit5-equivalent that allows parameterization at the class level (which is being handled by https://github.com/apache/iceberg/pull/9161)

Reply:
Ok, will hold off the changes for now

Reply:
I brought this up on the [DEV list](https://lists.apache.org/thread/5c4xs9w4jrot57dknq6vvwyg4o069kmo). Let's give people a few days and then we can come back with the right approach to properly migrate this.